api/controller.py

```diff
@@ -1915,6 +1915,10 @@ def fulfill_part_url(part):
                 content = etree.tostring(feed)
             status_code = 200
             headers["Content-Type"] = OPDSFeed.ACQUISITION_FEED_TYPE
+        elif fulfillment.content_link_redirect is True:
+            # Overdrive specific flow, the content link has no-drm linked to it
+            # So we should not be a proxy and instead redirect the client directly
```

Comment:
Would prefer that this comment be more generic. OD API is currently the only one using this flow, but it could be used for others, eventually.